codalab/lib/bundle_cli.py

```diff
@@ -1237,7 +1237,7 @@ def _worksheet_description(self, worksheet_info):
         fields = [
             ('Worksheet', self.worksheet_str(worksheet_info)),
             ('Title', formatting.verbose_contents_str(worksheet_info['title'])),
-            ('Tags', worksheet_info['tags']),
+            ('Tags', ', '.join(worksheet_info['tags'])),
```

Comment:
Yes (although we don't enforce it, but we should - add an issue?)  I think
it's pretty normal to have tags without spaces though.

On Fri, Feb 12, 2016 at 8:36 PM, Andrew <notifications@github.com> wrote:

> In codalab/lib/bundle_cli.py
> <https://github.com/codalab/codalab-cli/pull/334#discussion_r52822971>:
>
> > @@ -1237,7 +1237,7 @@ def _worksheet_description(self, worksheet_info):
> >          fields = [
> >              ('Worksheet', self.worksheet_str(worksheet_info)),
> >              ('Title', formatting.verbose_contents_str(worksheet_info['title'])),
> > -            ('Tags', worksheet_info['tags']),
> > +            ('Tags', ', '.join(worksheet_info['tags'])),
>
> Oh okay, is there no idea of multi-word tags in codalab?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/334/files#r52822971>.
>